src/Blocks/components/paragraph/paragraph-style.scss

```diff
@@ -1,5 +1,9 @@
 $paragraph: (
-	colors: global-settings(colors),
+	colors: (
+		primary: global-settings(colors, primary),
+		black: global-settings(colors, black),
+		muted: $base-text-muted-color
```

Comment:
all colors must go to the global colors manifest. If we define some in manifest and some in code it will become a mess very fast.